README.md

```diff
@@ -12,6 +12,7 @@ Our high level plan for the project can be found [here](https://docs.google.com/
 * Supports both **DSTU2** and **STU3** - the current version can be set in config, see [here](https://github.com/jembi/hearth/blob/master/config/default.json#L6)
 * Supports both **JSON** and **XML** with conversion between the two using the [FHIR.js module](https://www.npmjs.com/package/fhir). Supports using both the `Accepts:` header and the `_format` parameter for defining the response format.
 * Supports read, vread, search, create, update, delete, _history (on either resource types or specific resources, global history is not supported) and batch/transaction FHIR interactions for ALL resources
+* Supports creation of a resource through the update method for ALL resources. An update where the resource does not exist will create a new resource
```

Comment:
@bradsawadye @BMartinos Just a thought, this is not a standalone feature but a part of the update feature. Should we not perhaps have this documented here "`Supports read, vread, search, create, update, ...`"? It could be something like "`..., updated (or create it if new resource), ...`"

Reply:
@de-laz cool thats a good suggestion. We can add in the brackets explanation to the supports `update` part.

`supports ... ,update (or create if resource is not found)`

Reply:
@de-laz good pick. I've made the change